src/DependencyInjection/Configuration.php

```diff
@@ -494,6 +495,28 @@ private function addProxyClientSection(ArrayNodeDefinition $rootNode)
                             ->end()
                         ->end()
 
+                        ->arrayNode('cloudfront')
+                            ->children()
+                                ->scalarNode('distribution_id')
+                                    ->info('Identifier for your CloudFront distribution you want to purge the cache for')
+                                ->end()
+                                ->scalarNode('client')
+                                    ->info('AsyncAws\CloudFront\CloudFrontClient client to use')
+                                    ->defaultNull()
+                                ->end()
+                                ->variableNode('configuration')
+                                    ->defaultValue([])
+                                    ->info('Client configuration from https://async-aws.com/configuration.html')
+                                ->end()
+                            ->end()
+                            ->validate()
+                                ->ifTrue(function ($v) {
+                                    return !empty($v['client']) && !empty($v['configuration']);
```

Comment:
empty will not error if e.g. `$v` does not exist. this would be more explicit and should work too i think:
```suggestion
                                    return null !== $v['client'] && count($v['configuration']) > 0;
```